examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java

```diff
@@ -28,7 +28,23 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** End-to-end tests of BigQueryTornadoes. */
+/**
+ * An end-to-end tests of {@link org.apache.beam.examples.cookbook.BigQueryTornadoes}.
+ *
+ * <p>This test uses the data from an external source defined in {@link
```

Comment:
Temp location is only a place where temporary files are stored by the runner. I'm not sure that it has to be the same location as the BigQuery project. I'd skip the part where we mention `tempLocation` (it's described clearly in `PipelineOptions` interface), leaving it only in the running example and write something like this: 

```
This test reads the public samples of weather data from BigQuery, counts the number of tornadoes that occur 
in each month, and writes the results to BigQuery. It requires "BigQueryTornadoesIT" BigQuery dataset to be
created before running.

Running instructions:

<pre> ...
```

Reply:
Ok, I checked it and changed.